Specs/Renderer/TextureSpec.js

```diff
@@ -480,18 +454,18 @@ describe(
       }).contextToRender([0, 255, 0, 255]);
     });
 
-    it("draws the expected ETC1 compressed texture color", function () {
+    it("draws the expected KTX2 compressed texture color", function () {
       if (!context.etc1) {
         return;
       }
```

Comment:
This early exit is not relevant either. I assume this code can be deleted because uncompressed KTX2 don't require the WebAssembly module to decode them.